### ISSUE TITLE
revise annex A

### ISSUE THREAD

**Issue by tillmo** [tillmo/DOL#196]
annex A needs to be adapted to the new DOL ontology. It now captures the whole terms and definitions section, and not just logics and languages (LoLa), hence also the renaming into "DOL ontology".

**Comment by clange** [comment]
Re. renaming from "LoLa" to "DOL Ontology", I suggest the following new URLs. OK?

https://ontohub.org/meta/dol/ontology
http://www.omg.org/spec/DOL/Current/ontology

**Comment by tillmo** [comment]
OK!

**Comment by clange** [comment]
In connection to this discussion I changed some things in the ontology implementation; I hope these are consistent with OMG's workflows:

    Prefix: : <http://www.omg.org/spec/DOL/Current/ontology#>
    Ontology: <http://www.omg.org/spec/DOL/Current/ontology#> <http://www.omg.org/spec/DOL/20150706/ontology.owl>

The `dol.rdf` name that we had here for a while doesn't make too much sense.  The normative submission file was in RDF/XML, but it was an OWL ontology, and the standard filename extension for OWL ontologies encoded in RDF/XML is `.owl`.

Moreover in a mid-term perspective we'd like to implement the DOL ontology in DOL itself (as large parts of its semantics can't be expressed adequately in OWL), only making projections down to OWL available on request.  Technically this can be supported by a language-agnostic URI such as http://www.omg.org/spec/DOL/Current/ontology# above, from which concrete implementations in different languages can be served on request, based on HTTP content negotiation and specifying the `Content-type` header.

As we are, for now, only implementing the ontology in OWL, I kept a specific reference to OWL the `versionIRI`.
